08-wt-gradebook.Rmd

```diff
@@ -64,7 +82,9 @@ gradebook <- ExcelGradeBook
 
 Your environment will now have two versions of the dataset. There is *ExcelGradeBook* which was the original dataset imported. There is also *gradebook* which is currently a copy of *ExcelGradeBook*. As you progress through this section, we will work primarily with the *gradebook* dataset. Additionally, while working onward in this section of the book, if you make a mistake and mess up the *gradebook* data frame and are not able to fix it, you can reset the data frame to return to the same state as the original *ExcelGradeBook* data frame by running `gradebook <- ExcelGradeBook` again. This will overwrite any errors in the *gradebook* data frame with the originally imported *ExcelGradeBook* data frame. Afterwards, just continue running code from this point in the text.
 
-## Tidy Data
+## Process data
```

Comment:
does the distinction beneath this heading between `### Tidy Data` and `### Create new variables and further process the data` make sense?